bidscoin/plugins/dcm2niix2bids.py

```diff
@@ -226,6 +227,7 @@ def bidscoiner_plugin(session: Path, bidsmap: dict, bidsses: Path) -> Union[None
         scans_table.index.name = 'filename'
 
     # Process all the source files or run subfolders
+    bids_mappings: List[BidsMapping] = []
```

Comment:
I see what you are doing but I generally try to keep the plugins as short and simple as possible (to encourage external contributors to write plugiuns), and move stuff over as much as possible to bidscoiner. Building a list of mappings does add to the complexity...

Reply:
I think with the bidsmapping tsv file you were basically making a provenance table. For tracking provenance there is the W3C standard, and there is even a BEP in the making to implement this in BIDS:

https://github.com/bids-standard/BEP028_BIDSprov

That adds even more complexity, but would come with the benefit of being a standard...

Reply:
Yes, the idea was to quickly check after the bidscoiner run what was done - what source was converted to what BIDS.
Thank you for pointing me to the standards; it looks really useful.